src/main/java/org/openlmis/stockmanagement/dto/CalculatedStockOnHandDto.java

```diff
@@ -16,19 +16,37 @@
 package org.openlmis.stockmanagement.dto;
 
 import java.time.LocalDate;
+import java.util.UUID;
+
 import lombok.AllArgsConstructor;
-import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
-import org.openlmis.stockmanagement.domain.card.StockCard;
+import lombok.Setter;
+import lombok.ToString;
+import org.openlmis.stockmanagement.domain.event.CalculatedStockOnHand;
 
-@Data
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode(callSuper = true)
 @AllArgsConstructor
 @NoArgsConstructor
-public class CalculatedStockOnHandDto extends BaseDto {
+public final class CalculatedStockOnHandDto extends BaseDto
+        implements CalculatedStockOnHand.Exporter {
 
   private Integer stockOnHand;
 
-  private StockCard stockCard;
+  private UUID stockCardId;
```

Comment:
please change this to object reference ([schema](https://github.com/OpenLMIS/openlmis-stockmanagement/blob/master/src/main/resources/schemas/referenceObjectDto.json)) or remove it if this DTO will be part of Stock Card DTO.

Reply:
It will be part of Stock Card DTO, so I'm removing it.